packages/node_modules/@webex/webex-core/src/lib/services/interceptors/server-error.js

```diff
@@ -33,7 +33,7 @@ export default class ServerErrorInterceptor extends Interceptor {
           tags: {action: 'failed', error: reason.message, url: options.uri}
         });
 
-        return this.webex.internal.device.markUrlFailedAndGetNew(options.uri)
+        return Promise.resolve(this.webex.internal.services.markFailedUrl(options.uri))
           .then(() => Promise.reject(reason));
```

Comment:
Can you explain what's going on here?

Reply:
Yes, I am replacing the old method that it was calling [and didn't exist before this pr] with the new method. I re-implemented a deprecated method just in case a JS SDK user is still using that method.

The original method returned a `promise<string>`, the new method returns `string`.